### PR TITLE
Implement Trainer Navigation

### PR DESCRIPTION
**Pull request by pavle-doby**

### CODE REVIEW

File: src/modules/auth/Auth.router.tsx [hunk 1]
```diff
@@ -1,27 +1,17 @@
...
-import { Navigate, RouteObject } from 'react-router-dom';
-const authRouter: RouteObject = {
-  path: 'auth',
+import { RouteObject } from 'react-router-dom';
+
+export const authRouter: RouteObject = {
   children: [
     {
-      path: 'signup',
+      path: 'sing-up',
```
Comment (todor996) [inline src/modules/auth/Auth.router.tsx]: typo, sign-up

File: src/modules/auth/components/LogIn/LogInSection.tsx [hunk 2]
```diff
@@ -8,12 +9,10 @@ export function LogInSection(): JSX.Element {
 
         <label className="label">
           <span className="label-text">
-            You don't have an account?
-            <a href="#" className="cursor-pointer text-primary">
-              {' '}
-              Sign Up here
-            </a>
-            .
+            You don't have an account?{' '}
```
Comment (todor996) [inline src/modules/auth/components/LogIn/LogInSection.tsx]: same, /sign-up

File: src/modules/clients/trainer/Clients.router.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { lazy } from 'react';
+import { RouteObject } from 'react-router-dom';
```
Comment (todor996) [inline src/modules/clients/trainer/Clients.router.tsx]: mozda globalno da napravis Trainer router, da ne pises svuda trainer/ruta, nego tipa
TrainerRouter koji ce da ima subroutere

**Review by todor996 — CHANGES_REQUESTED** [review]

**Comment by pavle-doby** [comment]
In this PR there is a lot of experimenting... However, the `trainer-pov` module is the closest to a final structure/organization of files and code. Maybe a few tweaks in the code here and there and maybe we will remove the "pov" suffix (`.tpov`).

So, take a look, think about it and give me your thoughts. 😊

**Pull request merged**